src/fragments/lib/storage/js/get-properties.mdx

```diff
@@ -0,0 +1,49 @@
+Get the properties of the file using `getProperties` method.
+
+```typescript
+await Storage.getProperties(key: string);
+```
+
+The format of the response looks like this:
+
+```js
+ {
+    contentType: "image/jpeg",
+    contentLength: 6873,
+    eTag: "\"56b32cf4779ff6ca3ba3f2d455fa56a7\"",
+    lastModified: Wed Apr 19 2023 14:20:55 GMT-0700 (Pacific Daylight Time) {},
+    metadata: { owner : 'aws'}
+}
+
+```
+
+To utilize Encryption, the following options can be passed in to `Storage.getProperties`:
+
+```typescript
+const SSECustomerAlgorithm = 'string';
+const SSECustomerKey = new Buffer('...') || 'string';
+const SSECustomerKeyMD5 = 'string';
+
+const result = await Storage.getProperties(
+  { key: 'srcKey' },
+  {
+    SSECustomerAlgorithm,
+    SSECustomerKey,
+    SSECustomerKeyMD5
+  }
+);
+```
+
+### File Access Levels
+
+You can choose to configure [file access `level`](https://docs.amplify.aws/lib/storage/configureaccess/q/platform/js) as follows:
+
+```javascript
+
+const result = await Storage.get('welcome.png',  level?: private | protected | public);  // defaults to `public`
```

Comment:
as we are trying to show usage, types shouldn't been defined at the API call. Rather we can define a `level` variable with a union type of 'private' | 'protected' | 'public' and then assign a string value.

Reply:
And probably also changing the ```javascript  tag by ```typescript